src/backends/plonky2/signedpod.rs

```diff
@@ -61,6 +64,12 @@ pub struct SignedPod {
     pub dict: Dictionary,
 }
 
+#[derive(Serialize, Deserialize)]
+struct Data {
+    signature: String,
+    dict: Dictionary,
```

Comment:
The equivalent field for `MockSignedPod` is called `kvs`. I think it would be simpler for consumers if they had the same name, with the only difference being the pod_type and signature format.

Reply:
thanks, I've changed the field name :)